lib/decorators/focus.js

```diff
@@ -19,7 +19,7 @@ const delegated = {};
 function isFocusable(uri, path) {
   const group = get(uri, path);
 
-  return !group.schema[componentListProp] && !group.schema[componentProp];
+  return group ? !group.schema[componentListProp] && !group.schema[componentProp] : false;
```

Comment:
Maybe this will help with readability (Just a minor though)
```suggestion
  return group
  ? !(group.schema[componentListProp] && group.schema[componentProp])
  : false;
```

Reply:
I prefer it on the one line.